### DISCUSSION TITLE
blog/ambient-mesh-l4-traffic-path/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#334] [Announcements]
# blog/ambient-mesh-l4-traffic-path/

本文以图示和实际操作的形式详细介绍了 Ambient Mesh 中的透明流量劫持和四层（L4）流量路径。

https://jimmysong.io/blog/ambient-mesh-l4-traffic-path/

**Comment by Deancup** [comment]
检查 Ztunnel B Pod 上的路由规则,   这一节中提到，“进入 Ztunnel B Pod，使用 ip -d a 命令检查它的网卡信息，你会看到有一个 pistioout 网卡，它的 IP 为 192.168.127.2，这正是与 istioin 虚拟网卡建立的 Geneve 隧道的远端”，按照前面的描述，这里应该是与istioout虚拟网卡建立的Geneve隧道的远端吧

**Reply by rootsongjc** [reply]
感谢你的回复，看来你阅读的很仔细，这里确实是一个笔误，我修改好了。
